docs/air-pool/create-airpool-erc1155.md

```diff
@@ -1,34 +1,38 @@
 ---
-sidebar_position: 4
+sidebar_position: 5
 ---
 
-# Creating ERC1155 Air Pool
+# Creating ERC1155 AirPool
 
 This article focuses on how to create your first ERC 1155 reward pool on AirLyft.
 
-- Login to AirLyft and once you are on the dashboard, please click on AirPool icon as shown in the screenshot below.
+- There are two ways to create ERC 1155 AirPool. You can either create them from the AirPool tab, or you can click on **Create New Pool** while adding a reward to your campaign.
 
-![AirPool Dashboard](../images/createErc20pool.png)
+- In order to create AirPool from the Rewards, go to the Rewards tab under your campaign, and choose NFT Giveaway(ERC1155) and then choose AirPool Giveaway.
 
-- Next, you will be on AirPool page on which you can click on Create Pool option to create your first ERC1155 Pool.
+    ![AirPool from Rewards Tab](../images/rewardstab.png)
 
-![create pool](../images/createpool.png)
+- Click on the **Select an AirPool** dropdown, and click on **Create New Pool**.
 
-- Next step is for you to select the pool type where you will be provided with three options ERC20/XC20, NFT ERC721, and NFT ERC1155. Select NFT ERC 1155 from the list. 
+    ![Create New Pool](../images/createnewpool.png)
 
-![ERC 1155 Pool select](../images/erc1155select.png)
+- You can also go to the AirPool page where you can find your existing Pools or create a new one. Lets create a new AirPool by clicking on the Create AirPool button available on top right corner of the screen.
+
+    ![create pool](../images/airpoolcreate.png)
+
+- A new window popsup asking you to select the type of AirPool you want to create. Select Token ERC1155 from the list of options.
 
 - You will now be provided with few options to fill about the reward pool that you want to create such as Pool Name, Blockchain, Asset, Amount of Tokens to be locked.
 
-![Pool Details erc 1155](../images/erc1155pools.png)
+    ![Pool Details ERC1155](../images/erc1155pooldetail.png)
 
-- Next, if you don't see your tokens available on the list, you can find the [Request Asset Whitelist](https://docs.google.com/forms/d/e/1FAIpQLSdmdE3BmNwWQ1kZbKZqFzzRoBX38ltecXiSjuS5VEthwH28Yw/viewform) option. Click on it and fill up the short form for us to manually add the tokens to the asset list. 
+- Next, if you don't see your tokens available on the list, you can find the [Request Asset Whitelist](https://docs.google.com/forms/d/e/1FAIpQLSdmdE3BmNwWQ1kZbKZqFzzRoBX38ltecXiSjuS5VEthwH28Yw/viewform) option. Click on it and fill up the short form for us to manually add the tokens to the asset list.
```

Comment:
Also say "you can contact our support on "[support@airlyft.freshdesk.com](mailto:support@airlyft.freshdesk.com)" for a faster resolution.
Also - can you please replace all "support@kyte.one" email addresses (throughout the whole project to "support@airlyft.freshdesk.com"